### PR TITLE
Gluster HCI validations

### PR DESCRIPTION
**Pull request by sac**

### CODE REVIEW

File: roles/gluster_hci/README.md [hunk 1]
```diff
@@ -22,7 +22,8 @@ Role Variables
 | gluster_features_hci_port || 24007 | The port to be set for the remote  |
 | gluster_features_fqdn_check |true/false|true|Check if the hosts have valid FQDN. Set the variable to false to skip the FQDN check.|
 | gluster_features_min_disk ||15360|The minimum disk size that /var/log directory must have for the role to proceed. By default the role checks for 15G. Should be in megabytes|
-| gluster_features_force_diskcheck |true/false|true| Force the role to enforce the minimum disk size requirement for /var/log. If set to false, disk size check is ignored.|
+| gluster_features_force_diskcheck |true/false|true|Force the role to enforce the minimum disk size requirement for /var/log. If set to false, disk size check is ignored.|
```
Comment (sabose) [inline roles/gluster_hci/README.md]: variable name is misleading..suggests that the check is about disks

File: roles/gluster_hci/tasks/prerequisites.yml [hunk 2]
```diff
@@ -55,7 +55,26 @@
...
+        gluster_features_512B_check|default(true)
+  with_items: "{{ gluster_infra_volume_groups }}"
+
+- name: Check if disks have logical block size (VDO disks) of 512B else fail
+  command: cat /sys/block/{{item.device|basename}}/queue/logical_block_size
+  register: logical_blk_size
+  failed_when: logical_blk_size.stdout|int != 512
+  when: gluster_infra_vdo is defined and
+        gluster_features_512B_check|default(true)
+  with_items: "{{ gluster_infra_vdo }}"
```
Comment (sabose) [inline roles/gluster_hci/tasks/prerequisites.yml]: if vdo is used on the disks, is the disk check skipped? that's what we would want

Comment (sac) [reply roles/gluster_hci/tasks/prerequisites.yml]: For both vdo and non-vdo the logical block size is checked for the underlying disks. Do you mean that if we are using vdo we shouldn't be checking logical block size?

File: roles/gluster_hci/README.md (hunk 1 above)
Comment (sac) [reply roles/gluster_hci/README.md]: Ack! Will rename it to gluster_features_foce_varsizecheck

**Pull request merged**